server/src/main/scala/controllers/Application.scala

```diff
@@ -108,6 +108,24 @@ class Application @Inject()(cc: ControllerComponents, ctrl: DrtSystemInterface)(
 
   log.info(s"timezone: ${Calendar.getInstance().getTimeZone}")
 
+  def shouldUserViewBanner: Action[AnyContent] = Action.async { implicit request =>
+    val userEmail = request.headers.get("X-Auth-Email").getOrElse("Unknown")
+    val feedbackExistF: Future[Boolean] = ctrl.userFeedbackService.selectByEmail(userEmail).map(_.nonEmpty)
+    val bannerClosedAtF: Future[Option[Timestamp]] = ctrl.userService.selectUser(userEmail.trim).map(_.flatMap(_.feedback_banner_closed_at))
+    for {
+      feedbackExist <- feedbackExistF
+      bannerClosedAt <- bannerClosedAtF
+    } yield (feedbackExist, bannerClosedAt) match {
+      case (true, _) =>
```

Comment:
Can we check this logic? I thought a user would see the banner again after a couple of months of submitting feedback